### PR TITLE
Feature/update dependencies

### PR DESCRIPTION
**Pull request by davidalbela**
Update contracts to Solidity `v0.5.0` and update nodeJs dependencies:
- Update node dependencies.
- Update truffle configuration with new HDWalletProvider.
- Update Solidity to `v0.5.0`.
- Update Rinkeby contract address.
- Update Travis CI file.

### CODE REVIEW

File: truffle.js [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
+const mnemonic = process.env.MNEMONIC
 
 module.exports = {
   // See <http://truffleframework.com/docs/advanced/configuration>
   // to customize your Truffle configuration!
   networks: {
     development: {
       host: "127.0.0.1",
-      port: 9545,
+      port: 8545,
```
Comment (cag) [inline truffle.js]: Huh weird, I wonder why I had this...

Anyway, +1 to changing it back to the standard port.

File: contracts/TokenSplitter.sol [hunk 2]
```diff
@@ -1,10 +1,16 @@
-pragma solidity ^0.4.24;
+pragma solidity >=0.5.0 <0.7.0;
 
 import "@gnosis.pm/util-contracts/contracts/Token.sol";
 
 
 contract TokenSplitter {
-    function splitTokens(address[] _recipients, address _tokenAddress, uint _amountForEach) public {
+
+    uint overflowed = 2**32;
```
Comment (cag) [inline contracts/TokenSplitter.sol]: Why the overflow limit?

File: contracts/TokenSplitter.sol [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+
+    function splitTokens(address[] memory _recipients, address _tokenAddress, uint _amountForEach) public {
+        require(_recipients.length < 21, "TokenSplitter: only 20 addresses allowed");
```
Comment (cag) [inline contracts/TokenSplitter.sol]: Why only 20 addresses max?

**Comment by davidalbela** [comment]
> In general, I'd be fine with updating dependencies, except this PR does more than that. I'm not against the changes per se, but I don't see a reason necessarily for adding them either.
> 
> It seems a recipient cap would be enforced by gas limits anyway, so I don't see a reason to limit this to 20 per transaction if the blocks can accommodate more.
> 
> As for the overflow limit, 2**32 would be way too low anyway (that's about 4.3 GWei). Why not let the ERC20 token itself enforce that limit? It would have to keep track of balances anyway.

Ok, removed the `require` functions from contract. Sorry, I should not add that to that commit neither the PR.

Updated Travis CI file as well :)

File: .travis.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-- '9'
+- '13'
 cache:
   directories:
     - node_modules
 before_script: 
   - truffle compile
 script:
-  - ganache-cli -d -p 9545 --defaultBalanceEther 10000 -a 30 > /dev/null &
+  - ganache-cli -d -p 8545 --defaultBalanceEther 10000 -a 30 > /dev/null &
```
Comment (cag) [inline .travis.yml]: Could even remove `-p 8545` now since that's the default, but anyway that's not really a big deal.

**Pull request merged**